contrib/test.sh

```diff
@@ -3,7 +3,7 @@
 set -ex
 
 FEATURES="std alloc serde"
-MSRV="1\.48\.0"
+MSRV="1\.63\.0"
```

Comment:
SSOT break. This should just read `Cargo.toml`.

Reply:
If #106 gets support this doesn't matter because the file is deleted, can we leave it for now?

Reply:
OK, concept ACK doing #106